### PR TITLE
refactor(console): inline the search filter in the user table

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

refactor(console): inline search filter in the user table

### Before
image

### After
image

### CODE REVIEW

File: packages/console/src/components/AuditLogTable/index.tsx [hunk 1]
```diff
@@ -86,58 +86,60 @@ const AuditLogTable = ({ userId }: Props) => {
...
-                )}
-                <td>
-                  {payload.applicationId ? (
-                    <ApplicationName applicationId={payload.applicationId} />
-                  ) : (
-                    '-'
-                  )}
-                </td>
-                <td>{new Date(createdAt).toLocaleString()}</td>
+      <div className={resourcesStyles.table}>
```
Comment (charIeszhao) [inline packages/console/src/components/AuditLogTable/index.tsx:89]: Why is this stylesheet named `resourcesStyles`?

File: packages/console/src/pages/Users/index.tsx [hunk 2]
```diff
@@ -75,83 +77,88 @@ const Users = () => {
...
+          <table className={classNames(styles.table, !data && tableStyles.empty)}>
+            <colgroup>
+              <col className={styles.userName} />
+              <col />
+              <col />
+            </colgroup>
+            <thead>
+              <tr>
+                <th colSpan={userTableColumn} className={styles.filter}>
+                  <Search
```
Comment (wangsijie) [inline packages/console/src/pages/Users/index.tsx:92]: Per offline discussion, the filter should not be inside `table`

**Review by wangsijie — CHANGES_REQUESTED** [review]

File: packages/console/src/pages/Users/index.tsx [hunk 3]
```diff
@@ -75,83 +77,88 @@ const Users = () => {
...
+              {!data && error && (
+                <TableError
+                  columns={3}
+                  content={error.body?.message ?? error.message}
+                  onRetry={async () => mutate(undefined, true)}
+                />
+              )}
+              {isLoading && <TableLoading columns={3} />}
+              {users?.length === 0 && (
+                <TableEmpty columns={3}>
```
Comment (charIeszhao) [inline packages/console/src/pages/Users/index.tsx:120]: Maybe also replace these `column={3}` with `column={userTableColumn}`?

Comment (xiaoyijun) [reply packages/console/src/pages/Users/index.tsx]: Cool, updated.

File: packages/console/src/pages/Users/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/pages/Users/index.tsx]: Updated.

File: packages/console/src/components/AuditLogTable/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/AuditLogTable/index.tsx]: We treat applications, API resources, audit logs, and users as `resources`, and the `resourcesStyles` is shared among these `resources` pages.

**Review by wangsijie — APPROVED** [review]

**Pull request merged**